.travis/install_docs.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 set -ev
 
-pip install -r docs/requirements.txt --user
+pip install -r Resources/doc/requirements.txt --user
```

Comment:
this change reverts a change of #502 - @core23 can you please check the devkit and make it not break the doc build?

Reply:
https://github.com/sonata-project/dev-kit/pull/282 covers the problem